charts/schema-registry/values.yaml

```diff
@@ -15,8 +15,8 @@
 replicas: 2
 
 image:
-  repository: pravega/schema-registry
-  tag: 0.0.1
+  repository: pravega/schemaregistry
```

Comment:
same comment here. i think we may need to do a 0.1.1 release for schema registry with auth string changes so that we can utilize the new key cloak auth handler implementation from pravega.